src/core/utils.py

```diff
@@ -0,0 +1,22 @@
+from src.core.common_path import CommonPaths
+
+
+class ConfigLoader:
+    @staticmethod
+    def load_application_properties(config_name="app.properties") -> dict:
+        """
+        Load application properties.
+        :param config_name:
+        :return: dict of properties
+        """
+        resources_dir_path = CommonPaths.project_root
+        file_path = resources_dir_path.joinpath(config_name)
+        properties = {}
+        with open(file_path, "r", encoding="utf-8") as file:
+            for line in file:
+                line = line.strip()
+                if not line or line.startswith("#"):
+                    continue
+                key, value = line.split("=", 1)
+                properties[key.strip()] = value.strip()
+        return properties
```

Comment:
@vasylboliuk since we are trying to follow 12 factor app and its configuration management pillar, I believe there would be different sources for the settings as we grow. It could be a yaml file, env variables, dotenv files, secret files, secret manager, configuration manager (AWS SSM as an example), etc. It looks to me more preferable to have a single source of truth for different configuration sources and change their priorities as we desire. I would suggest to look into pydantic BaseSettings class, its customize_sources method and PydanticBaseSettings source from which you could inherit to implement your own settings source (yaml file as an example) -> https://docs.pydantic.dev/latest/concepts/pydantic_settings/

Reply:
Fixed.